onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -621,6 +621,60 @@ TEST(MathOpTest, Pow_Broadcast_Scalar1) {
   test.Run();
 }
 
+TEST(MathOpTest, Pow_float_int64) {
+  OpTester test("Pow", 12);
```

Comment:
Can you add some tests with broadcasting? Or run some of the existing tests with both the original opset and opset 12?